pages/terra/policy.mdx

```diff
@@ -123,7 +237,7 @@ Example:
 %cargo_install
 ```
 
-### Go
+#### Go
 
 Use the `go` compiler directly instead of using Fedora's Go macros, as this will generate offline dependencies that we do not want to package.
 Alternative, use the `_online` macro variants:
```

Comment:
Write an example for using the macro variants, in case someone is dense enough to not figure out what it's actually called

Reply:
typo
```suggestion
Alternatively, use the `_online` macro variants:
```